packages/edit-widgets/src/components/widget-area/style.scss

```diff
@@ -1,6 +1,18 @@
 .edit-widgets-widget-area {
-	max-width: $content-width;
+	max-width: $widget-area-width;
 	margin: 0 auto 30px;
+
+	// Reduce padding inside widget areas
+	.block-editor-block-list__layout {
+		padding-left: 0;
+		padding-right: 0;
+	}
+	// By default the default block appender inserter has a negative position,
+	// but given that on the widget screen we have 0 padding we need to remove the negative position.
+	.block-editor-default-block-appender .block-editor-inserter,
+	.block-editor-block-list__empty-block-inserter {
```

Comment:
I think these can be nested, do we want that config for all the nested appenders?

Reply:
Hi @youknowriad, nested appenders have a CSS rule with much higher specificity that overwrites this value. Ther value was already different between the top-level appenders and nested ones in the edit-post.